abis_mapping/templates/incidental_occurrence_data_v3/mapping.py

```diff
@@ -1441,6 +1441,11 @@ def add_sampling_field(
         graph.add((geometry_node, a, utils.namespaces.GEO.Geometry))
         graph.add((geometry_node, utils.namespaces.GEO.asWKT, geometry.to_transformed_crs_rdf_literal()))
 
+        spatial_accuracy = row.get("coordinateUncertaintyInMeters")
```

Comment:
you're sure of that @joecrowleygaia ? I prefer to deal with `None` that with an exception.  
Open discussion.

Reply:
but we've done with direct key access so far. So let be consistent.

Reply:
After reflection, you're right @joecrowleygaia. Better to use direct key access to early detect schema changes.

Reply:
I'll move it back to the way it was before it was suggested I use `get` in a previous comment.

Reply:
If that is relating to the one i suggested changing to `get`, it was slightly different since it was accessing a field that didn't exist on the template. Sorry for the confusion.